README.md

```diff
@@ -1,24 +1,200 @@
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Configuration
+<!-- PROJECT DESCRIPTION -->
 
-* Database creation
+# 📖 [Hello Rails React] <a name="about-project"></a>
 
-* Database initialization
+**[Hello Rails React]** is an app with a rails backend with a react frontent in the same repo.
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### React <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<details>
+  <summary>Rails</summary>
+  <ul>
+    <li><a href="https://rails.org/">Rails</a></li>
+  </ul>
+</details>
 
-* ...
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Shows random greeting from the database]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Live Demo Link is not yet available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- An IDE
+- Node.js
+### Setup
+
+Clone this repository to your desired folder:
+```
+git clone https://github.com/Reggeditt/hello-rails-react.git
+```
+### Install
+
+Install this project with:
+```
+bundle install
+
+rails db:create
+
+rails db:migrate
+
+rails db:seed
+
+```
+### Usage
+
+To run the project, execute the following command:
+
+```bash
+  ./bin/dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm test
+```
+### Deployment
+
+You can deploy this project using:
+<!-- 
+```sh
+  npm run deploy
+```
+ -->
+ not available for deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kodjo-Sarso Tetteh**
+
+- GitHub: [@Reggeditt](https://github.com/Reggeditt)
+- Twitter: [@tksarso](https://twitter.com/tksarso)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tetteh-kodjo-sarso-888989125/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+<!-- 
+- [ ] **[Api fetch data]** -->
+<!-- - [ ] **[new_feature_2]** -->
+<!-- - [ ] **[new_feature_3]** -->
```

Comment:
- [ ] Your `Future feature` section is empty and as required, this is a required section and needs to be filled up. you need to add at least one feature. here's a suggestion `Add a new greeting, Select language to get greeting`  :+1: :100:

Reply:
thanks